paymentsheet/src/main/java/com/stripe/android/link/LinkControllerViewModel.kt

```diff
@@ -211,15 +211,10 @@ internal class LinkControllerViewModel @Inject constructor(
         onSuccess: suspend (LinkConfiguration) -> Unit
     ) {
         val configuration = requireConfiguration()
-            .map {
-                it.copy(
-                    customerInfo = LinkConfiguration.CustomerInfo(
-                        name = null,
-                        email = email,
-                        phone = null,
-                        billingCountryCode = null,
-                    )
-                )
+            .map { config ->
+                email
+                    ?.let { config.copy(customerInfo = config.customerInfo.copy(email = email)) }
```

Comment:
can we have a conflict between the email passed in the `LinkController` methods and the one in the default billing details?

Reply:
Yeah we can. Not sure if what the expected behavior should be.